playbooks/common/openshift-cluster/config.yml

```diff
@@ -31,6 +31,6 @@
 
 - include: ../openshift-master/config.yml
 
-- include: additional_config.yml
-
 - include: ../openshift-node/config.yml
+
+- include: additional_config.yml
```

Comment:
I'm okay with separating router/registry/etc from additional_config but what is nice about running additional config before nodes?